proto/viam/app/v1/app.proto

```diff
@@ -508,8 +508,34 @@ message Fragment {
   string organization_owner = 4 [(tagger.v1.tags) = "bson:\"organization_owner\" json:\"owner\""];
   bool public = 5 [(tagger.v1.tags) = "bson:\"public\" json:\"public\""];
   google.protobuf.Timestamp created_on = 6 [(tagger.v1.tags) = "bson:\"created_on\""];
+  string organization_name = 7;
```

Comment:
We don't have this in the db Fragment model, but we return it to the FE to be used so it should be in the definition.